src/main/java/komo/fraczek/toukparking/domain/ParkingCodeGenerator.java

```diff
@@ -0,0 +1,14 @@
+package komo.fraczek.toukparking.domain;
+
+
+import org.apache.commons.lang3.RandomStringUtils;
+
+public class ParkingCodeGenerator {
+
+    public static String getCode(){
+        String firstPart = RandomStringUtils.randomAlphabetic(3);
+        String secondPart = RandomStringUtils.randomNumeric(3);
+        return firstPart.toUpperCase() + "-" + secondPart;
```

Comment:
No to jest dosyć trywialne generowanie losowych wartości - co się stanie jak wygenerujesz duplikat?